### DISCUSSION TITLE
Open new dialog  Instead of retrive form

### DISCUSSION THREAD

**Discussion by hosnabigdeli** [serenity-is/Serenity#5838] [Help]
hi 
how can i click on grid and open new dialog instead retrive form

**Comment by minhhungit** [comment]
handle onCick method on grid, there is an example here https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/Modules/BasicSamples/Grids/InlineActionButtons/InlineActionGrid.ts#L46
